### ISSUE TITLE
Figure caption inside markdown environment with hybrid option allows italic but not bold

### ISSUE THREAD

**Issue by nopria** [Witiko/markdown#88]
The following MWE shows that caption allows emphasis (`*`) but not strong emphasis (`**`):

```
\documentclass{article}
\usepackage[utf8]{inputenc}
\usepackage[hybrid]{markdown}

\begin{document}

\begin{markdown}

# MWE bold in caption with markdown

\begin{figure}
    \centering
    \includegraphics{example-image}
    \caption{Caption with markdown *emphasis* inside.}
    \label{fig:my_label}
\end{figure}

\end{markdown}

\end{document}

```

**Comment by Witiko** [comment]
@nopria [The default definition of `\markdownRendererStrongEmphasis`](https://github.com/Witiko/markdown/blob/master/markdown.dtx#L18344) used to be fragile. It has been fixed in https://github.com/Witiko/markdown/commit/89a031a2ba54a7dbb42dc41562af78d9d8cd857a and will be released in Markdown v0.2.10.

For a quick fix, you can either redefine the strong emphasis renderer:
``` tex
\markdownSetup{
    renderer={
        strongEmphasis={\textbf{#1}}
    }
(
```
or you can add the `\protect` command:
```
\begin{figure}
    \centering
    \includegraphics{example-image}
    \caption{Caption with markdown \protect**strong emphasis** inside.}
    \label{fig:my_label}
\end{figure}
```

**Comment by Witiko** [comment]
Please, let me know if you had any other questions.
